opencontrol.yaml

```diff
@@ -0,0 +1,33 @@
+schema_version: "1.0.0"
```

Comment:
Does schema version even matter?

Reply:
I believe that it does since the schema versioning is part of OpenControl